graph/impl/src/main/java/org/jboss/windup/graph/GraphContextImpl.java

```diff
@@ -85,6 +87,7 @@ public GraphContextImpl create()
         FileUtils.deleteQuietly(graphDir.toFile());
         TitanGraph titan = initializeTitanGraph();
         initializeTitanIndexes(titan);
+        generateTypeScriptModels();
```

Comment:
Yes, this is WIP and as the comment of the class says, it will be moved to a RuleProvider or even (better) to a standalone Windup command.